index.html

```diff
@@ -8,12 +8,92 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
+ </head>
 <body>
 
+
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
-
+    <header class="header">
+        
+          <img src="img/karma-logo.svg" alt="image-logo" class="logo"/>
```

Comment:
I can see a number of tags that are not properly indented, including this one. Please make it consistent. See https://syllabus.codeyourfuture.io/guides/code-style-guide for more info on how to indent the codes automatically